packages/@lwc/ssr-runtime/src/render.ts

```diff
@@ -160,14 +165,22 @@ export async function serverSideRenderComponent(
             tagName,
             props,
             null,
+            null,
             null
         )) {
             markup += segment;
         }
     } else if (mode === 'async') {
-        await (generateMarkup as GenerateMarkupFnAsyncNoGen)(emit, tagName, props, null, null);
+        await (generateMarkup as GenerateMarkupFnAsyncNoGen)(
+            emit,
+            tagName,
+            props,
+            null,
+            null,
+            null
+        );
     } else if (mode === 'sync') {
-        (generateMarkup as GenerateMarkupFnSyncNoGen)(emit, tagName, props, null, null);
+        (generateMarkup as GenerateMarkupFnSyncNoGen)(emit, tagName, props, null, null, null);
```

Comment:
TODO: I think we're actually missing some nulls/undefineds here.

Reply:
Just make all the params optional and don't send them!

Reply:
They would technically be `undefined` then; not sure if it matters in this case. I also prefer explicit here anyway, and I expect we will just keep adding args here since this is generated code and it's better to be optimal than ergonomic.